src/strings/en-us.json

```diff
@@ -667,7 +667,7 @@
     "LabelEnableIP6": "Enable IPv6",
     "LabelEnableIP6Help": "Enable IPv6 functionality.",
     "LabelEnableLUFSScan": "Enable LUFS scan",
-    "LabelEnableLUFSScanHelp": "Enable LUFS scan for music (This will take longer and more resources).",
+    "LabelEnableLUFSScanHelp": "Clients can normalize audio playback to get equal loudness across tracks. This will make the scan longer and take more resources.",
```

Comment:
"The scans" is not clear. Maybe something like this works:

```suggestion
    "LabelEnableLUFSScanHelp": "Clients can normalize audio playback to get equal loudness across tracks. This will impact the duration of library scans resources used.",
```

Reply:
I will change "scans" to "library scans".

I disagree on the rest of the change, though, because it changes the language from simple to less simple. What do you think?

Reply:
yeah that's fine

Reply:
OK, great! In that case this is ready.